### PR TITLE
WIP: Nadine i18n for members and tablet + fr translation

### PR DESCRIPTION
**Pull request by manens**

### CODE REVIEW

**Comment by manens** [comment]
To create a new po file or update an existing one : 
`django-admin makemessages -i comlink -l fr`
`django-admin makemessages -i comlink -l de`

To compile the po files : 
`django-admin compilemessages`

File: member/templates/member/events/booking_confirm.html [hunk 1]
```diff
@@ -55,7 +56,7 @@ <h4 class='header-font' style='text-align:center; color:black;'>Confirm Booking<
                 {% if forloop.counter0|divisibleby:loop_count %}
                   <th class="rotate"><div><span>
                     {{ time.hour }}:{{ time.minutes }}
-                  </div></span></th>
```
Comment (manens) [inline member/templates/member/events/booking_confirm.html]: @jsayles Do you want to remove this from the PR ?

Comment (jsayles) [reply member/templates/member/events/booking_confirm.html]: No this is OK. Thank you.

**Comment by jsayles** [comment]
This has gotten too far from master.  I tried merging it with my translations branch which has been merged with master and there were too many conflicts to resolve cleanly.  Closing this and we can start fresh with a new PR when you are ready.

**Pull request closed without merging**